router/core/graphql_prehandler.go

```diff
@@ -23,13 +25,20 @@ import (
 	rtrace "github.com/wundergraph/cosmo/router/pkg/trace"
 
 	"github.com/go-chi/chi/v5/middleware"
+	"github.com/mazrean/formstream"
+	httpform "github.com/mazrean/formstream/http"
+	"github.com/pkg/errors"
 	"go.uber.org/zap"
 
 	"github.com/wundergraph/cosmo/router/internal/pool"
 
 	"github.com/wundergraph/graphql-go-tools/v2/pkg/engine/resolve"
 )
 
+const (
+	MaxSupportedFilesUpload = 10
```

Comment:
It would be great if we would support this as a config.

Reply:
Done